src/as.rs

```diff
@@ -18,7 +18,7 @@ impl<Frm: ?Sized, To: ?Sized> Default for Conv<Frm, To> {
 
 /// Trait performing the specialization.
 pub trait ExtractRef {
-    /// Inout reference type.
```

Comment:
Does `Inout` happen to be a technical term I don't know, or is `Input` correct?

Reply:
seems like a typo to me.